api/serializers.py

```diff
@@ -1,39 +1,71 @@
+from django.contrib.auth import authenticate
+from django.contrib.auth.models import update_last_login
 from rest_framework import serializers
 from rest_framework_jwt.settings import api_settings
-from django.contrib.auth.models import User
-from .models import Video
+
+# from django.contrib.auth.models import User

```

Comment:
I really dislike commented code like this. If the `User` model is not needed, why not completely remove it?